backend/index.js

```diff
@@ -516,7 +551,24 @@ app.post('/logout', (req, res) => {
   });
 });
 
-app.use('/uploads', express.static(path.join(process.cwd(), 'uploads')));
+//Endpoint to delete stories
+
+app.delete('/api/stories/expired', async (req, res) => {
```

Comment:
We may want to consider deleting stories from DB at fetch time if we notice any expired stories. Not sure if you may have included this further up the stack